week1/homework/src/server.js

```diff
@@ -2,16 +2,20 @@
 
 const http = require('http');
 
-/* `createServer` MUST return an instance of `http.Server` otherwise the tests
- * will fail.
- */
+const getResponse = (responseStatus, response, responseObj) => {
```

Comment:
it's probably a valid name from your point of view, but from mine I would prefer `sendResponse` since it's sending it, and no big deal 😄

Reply:
Sometimes while writing the code, the more difficult one is to pick correct function names, generally I am not lucky about this. :)) I thought like this while choosing the name.  'sending request' and 'getting response'